packages/theme-default/src/components/Nav/index.module.scss

```diff
@@ -58,7 +58,7 @@
 }
 
 @media (max-width: 768px) {
-  :root {
+  html:root {
```

Comment:
No idea why it does not work without higher priority at least on Microsoft Edage.